commands/keplr.js

```diff
@@ -60,13 +74,15 @@ const keplr = {
     extensionVersion = keplrExtensionData.version;
     registrationUrl = `chrome-extension://${extensionId}/register.html`;
     permissionsUrl = `chrome-extension://${extensionId}/popup.html#/setting/security/permission`;
+    popupUrl = `chrome-extension://${extensionId}/popup.html`;
```

Comment:
rename to 'dashbaordUrl' or 'homeUrl'. it more closely resemble the intended purpose of the page

Reply:
You're right. These names are more intention revealing. But I'm noticing that `popup.html` has dynamic content. For example, it's the same URL when we encounter a notification. I feel we should stick with `popupUrl` as a generic